website/docs/interact-with-data.mdx

```diff
@@ -0,0 +1,287 @@
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+
+# Interact with data
+The final step of getting started with ComposeDB is interacting with your data. In this guide you will learn how to perform GraphQL queries and mutations using your composite.
+
+<!-- >💡 Want to interact with data from JavaScript? See [Client configuration](https://www.notion.so/Client-configuration-6147b53d9e01478f91bd81194c4a39c2). -->
+
+## Setup
+### GraphQL Server
+To interact with data on the network, start a local GraphQL server by running the command below. Note that you have to provide your `did-private-key` here — it is also required for performing mutations, covered below.
+
+```sh
+composedb graphql:server --ceramic-url=http://localhost:7007 --graphiql runtime-composite.json --did-private-key=your_private_key --port=5005
+```
+
+>✏️ ***Note:*** You can customize the port by configuring the `—-port` flag.
+
+
+The output will display a URL, for example:
+```sh
+GraphQL server is listening on http://localhost:5005/graphql
+```
+
+### GraphQL Web UI
+In your browser, visit the URL that your local GraphQL server is listening on. You will see a simple UI that you can use to easily interact with your data:
+![GraphQL Web UI](../static/img/data-model-table.png)
```

Comment:
See https://docusaurus.io/docs/markdown-features/assets#static-assets for using static assets

```suggestion
![GraphQL Web UI](/img/data-model-table.png)
```